src/backend/rest.ts

```diff
@@ -68,6 +68,7 @@ export default class RestApiBackend implements Backend {
   async testDatasource(): Promise<unknown> {
     // TODO: fix next line
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    // TODO: Maybe test for used endpoints directly instead of version?
```

Comment:
this comment is introduced with commit "REST-API testDatasource will now actually test for endpoints to be ailable". but the comment and the commit message do not match?